notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaNotificationDao.java

```diff
@@ -52,6 +52,7 @@
      * persistence context, it is returned;  otherwise, null is returned.
      */
     @Override
+	@Transactional
```

Comment:
should probably specify readOnly=true. Also spacing issue. Insure your IDE is set to use 4 spaces not tabs.